### PR TITLE
fix: md format

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 1]
```diff
@@ -311,10 +311,39 @@ Item {
             textFormat: TextEdit.RichText
             text: {
                 if(contentType === Constants.stickerType) return "";
+                let msg = linkify(message);
                 if(isEmoji){
-                    return Emoji.parse(message, "72x72");
+                    return Emoji.parse(msg, "72x72");
                 } else {
-                    return Emoji.parse(linkify(message), "26x26");
+                    return `<html>
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:318]: Have you tried putting the `<html>` and `<style>` in a higher up component? That way, we wouldn't declare them for each message

Comment (richard-ramos) [reply ui/app/AppLayouts/Chat/ChatColumn/Message.qml:318]: I tried but it did not work :(

Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatColumn/Message.qml:318]: Ugh, that sucks. I guess QML has some kind of sandboxing for each component. 

I'm not sure what to do. It's just that this solution is super mega ugly. I'll try to think of something

Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatColumn/Message.qml:318]: Welp, I searched and tested on your branch and I can't find another possibility for now, so I'll approve

**Pull request merged**